deployment_report/model/utils/config_util.py

```diff
@@ -29,22 +30,40 @@
 _DATABASE_HOST_KEY_ = "DATABASE_HOST"
 _DATABASE_NAME_KEY_ = "DATABASE_NAME"
 _DATABASE_PORT_KEY_ = "DATABASE_PORT"
+_DATABASE_SSL_ENABLED_KEY_ = "DATABASE_SSL_ENABLED"
 _EXTERNAL_DATABASE_KEY_ = "EXTERNAL_DATABASE"
 _DATASOURCE_USERNAME_KEY_ = "SPRING_DATASOURCE_USERNAME"
 
+# SAS postgres pgcluster keys
+_PGCLUSTER_HOST_KEY_ = "host"
+_PGCLUSTER_PORT_KEY_ = "port"
+_PGCLUSTER_NAME_KEY_ = "database"
+_PGCLUSTER_SSL_KEY_ = "ssl"
+_PGCLUSTER_CONNECT_KEY_ = "connection"
+
 # database info keys
-_HOST_KEY_ = "host"
-_NAME_KEY_ = "name"
-_PORT_KEY_ = "port"
-_TYPE_KEY_ = "type"
-_USER_KEY_ = "user"
+_HOST_KEY_ = "Host"
+_NAME_KEY_ = "Name"
+_PORT_KEY_ = "Port"
+_TYPE_KEY_ = "Type"
+_SSLENABLED_KEY_ = "SSL_Enabled"
```

Comment:
Keys in the data dictionary are camel-cased by convention, formatting for display should be done in the jinja template.

Reply:
Add new class DatabaseDetails to viya_deployment_report_keys.py.